package.json

```diff
@@ -217,7 +217,7 @@
   "scripts": {
     "dependencies": "node scripts/bundle-and-gitignore-deps.js && node scripts/dependency-graph.js",
     "dumpconf": "env | grep npm | sort | uniq",
-    "preversion": "bash scripts/update-authors.sh && git add AUTHORS && git commit -m \"chore: update AUTHORS\" || true",
+    "authors": "bash scripts/update-authors.sh",
```

Comment:
The action takes care of committing anything that gets touched via `npm run rp-pull-request` so we don't need to commit it here anymore.

Reply:
I also renamed this from `preversion` since we won't manually be running `npm version` anymore for releases. Since it gets called from an action I opted not to name it anything prefixed with `pre` or `post` since that might signal it gets called automatically.